client.go

```diff
@@ -489,17 +489,32 @@ func (c *Client) websocket(operation string, secret string) (*websocket.Conn, er
 }
 
 func (c *Client) url(elem ...string) string {
+	// Normalize the URL
 	path := strings.Join(elem, "/")
+	entries := []string{}
```

Comment:
What about stripping slashes from elem prior to joining?  This would be especially nice if there is a built-in function for transforming slices.

```
entries := []string{}
for entry := range elem {
	append(entries, strings.Replace(entry, "/", "", -1))
}
path := strings.Join(entries, "/")
```